cmd/controller/pkg_repository_reconciler.go

```diff
@@ -5,20 +5,24 @@ package controller
 
 import (
 	"context"
-
-	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
-	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/reftracker"
+	"fmt"
 
 	"github.com/go-logr/logr"
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
+	v1alpha12 "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/packaging/v1alpha1"
```

Comment:
probably better to name this pkgv1alpha1.

Reply:
Added